test/lumo/repl_tests.cljs

```diff
@@ -67,3 +69,10 @@
 (deftest test-root-resource
   (is (= (lumo/root-resource 'foo-bar-baz) "/foo_bar_baz"))
   (is (= (lumo/root-resource 'foo.bar.baz) "/foo/bar/baz")))
+
+(deftest-with-env test-get-arglists @(cljs/empty-state)
+  (is (= (lumo/get-arglists "cljs.core/map") '([f] [f coll] [f c1 c2] [f c1 c2 c3] [f c1 c2 c3 & colls])))
+  (is (= (lumo/get-arglists "map") '([f] [f coll] [f c1 c2] [f c1 c2 c3] [f c1 c2 c3 & colls])))
+  (is (= (lumo/get-arglists "clojure.core/when") '([test & body])))
+  ;; (is (= (lumo/get-arglists "when") '([test & body]))) ;; cannot test it calling lumo.repl/init works in tests
```

Comment:
Just so that I don't forget, the problem here was that we don't have this key for `when` in the compiler env unless we are able to load the compiler cache (call to `lumo.repl/init`). We cannot load the compiler cache because `lumo.repl/init` triggers a `$$LUMO_GLOBALS.load is not a function`.